test/fixtures/default/variables.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "credentials_path" {}
+
+variable "name" {
+  default = "pf-test-integration"
+}
+
+variable "random_project_id" {
+  default = "true"
+}
+
+variable "org_id" {}
+
+variable "folder_id" {
+  default = ""
+}
+
+variable "domain" {
+}
+
+variable "usage_bucket_name" {
```

Comment:
If we create this bucket as a fixture resource created within the test module, we can drop this variable.

Reply:
Yes, that's a good idea (assuming it can be created in the same project).